### PR TITLE
Add google map, make click to map update App state location, add func…

### PR DESCRIPTION
**Pull request by TravisPak**
…tion to find user location:

### CODE REVIEW

**Review by ZloNAGit — APPROVED** [review]
👍  Nice work, Travis. If we're not looking to keep the commented console logs/imports, we should probably remove them before merging, but besides that it looks good to me. I'm excited to see how it's rendering out.

**Comment by egrochowski** [comment]
Thomas covered all I had to say. Looks good.

**Review by egrochowski — APPROVED** [review]
Thomas covered all I had to say. Looks good.

File: client/components/App.jsx [hunk 1]
```diff
@@ -1,18 +1,57 @@
...
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
+
+const App = () => {
+  const [state, setState] = useState ({
+    location: {lat: 39, lng: -94},
+    searchBoxText: 'Search a City!'
+  })
```
Comment (DedotadedWam) [inline client/components/App.jsx:10]: is location referencing current location or the eventual location in the searchBoxText? If it is the former, might want to give searchBoxText its own useState and give location a more descriptive title.

Comment (DedotadedWam) [reply client/components/App.jsx:10]: based off of the rest of the component looks like it is the former of the 2 scenarios provided.

File: client/components/GoogleMap.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <div style={{ height: '80vh', width: '90%', margin: '10vh auto 10vh auto'}}>
+        <GoogleMapReact
+          bootstrapURLKeys={{ key: GOOGLE_API_KEY}}
+          defaultCenter={{lat: 39, lng: -94}}
+          defaultZoom={8}
+          onClick={mapClick}
+        >
+          <Marker lat={props.location.lat} lng={props.location.lng} />
+        </GoogleMapReact>
+
```
Comment (DedotadedWam) [inline client/components/GoogleMap.jsx:30]: any way to add dark theme to the map?

**Review by DedotadedWam — APPROVED** [review]

**Pull request merged**